rfc/eol_kubernetes_v1.md

```diff
@@ -0,0 +1,109 @@
+  - [x] Remove the V1 provider codelab ([spinnaker.github.io/pull/1586](https://github.com/spinnaker/spinnaker.github.io/pull/1586)). Add a redirect from the V1 codelab to the V2 codelab ([spinnaker.github.io/pull/1649](https://github.com/spinnaker/spinnaker.github.io/pull/1649)).
+  - [x] Remove inbound links to V1 provider setup ([spinnaker.github.io/pull/1649](https://github.com/spinnaker/spinnaker.github.io/pull/1649)).
+  
+- [] Publish a blog post on the Spinnaker Community Blog. This post will answer the following questions:
+  - [] Why is Spinnaker removing the V1 provider?
+  - [] When is Spinnaker removing the V1 provider? What is the latest supported release I will be able to use with the V1 provider enabled?
+  - [] How can I successfully execute a zero-downtime migration from the V1 provider to the V2 provider? What are some first-hand lessons from teams who have done this?
+
+- [] Audit existing Spinnaker content to update.
+  - [] Ensure all guides on spinnaker.io reference V2 only.
+  - [] Standardize how we refer to the V2 provider (currently, mix of “V2” and “Manifest Based”) and the V1 provider (currently, mix of “V1” and “Legacy”).
+  - [] Add documentation to spinnaker.io on why and when the V1 provider will be / was removed.
+  - [] Add an addendum to any blog posts on the Spinnaker Community Blog that reference the V1 provider highlighting that it is no longer supported.
+  - [] Compile a list of Armory documentation and blog posts for Armory to update.
+  
+- [] Ensure clear and consistent communication of the plan:
+  - [] Highlight the upcoming removal in each release’s curated changelog leading up to the release in which V1 will be officially removed.
+  - [] Post about the RFC in relevant Slack channels before each release: #dev, #kubernetes, #sig-kubernetes.
+  - [] Post about the RFC in any new V1-related issues that are opened on GitHub.
+  - [] Add helpful warnings to Clouddriver, Halyard, and Deck in the releases leading up to the removal.
+  - [] Consider requiring users to enable a disabled-by-default feature flag (Halyard, Clouddriver) in releases leading up to the removal of V1 in order to deploy with one or more enabled V1 accounts.
+
+- [] Remove V1-only code.
+  - [] Audit each microservice to ensure no V2 code paths exercise V1 code.
+    Note: ezimanyi@ and I have already ensured isolation between V1-only and V2-only code in Clouddriver ([clouddriver/pull/4024](https://github.com/spinnaker/clouddriver/pull/4024), [clouddriver/pull/4028](https://github.com/spinnaker/clouddriver/pull/4028)) and Deck ([deck/pull/7451](https://github.com/spinnaker/deck/pull/7451)).
+ 
+## Drawbacks
+
+- We will be taking away a feature set that some community members currently rely on for critical delivery workflows.
+  Migrating from the V1 to the V2 provider will be a significant lift for Spinnaker operators. 
```

Comment:
I agree, even with a zero downtime migration (like you mentioned above) I suspect there are some operational burdens that these teams will need to accept along with the migration. If they aren't already doing things with manifests they'll have to start.